great_expectations/data_context/cloud_migrator.py

```diff
@@ -299,9 +303,37 @@ def _send_configuration_bundle(
         configuration_bundle: ConfigurationBundle,
         serializer: ConfigurationBundleJsonSerializer,
     ) -> AnyPayload:
-        # Serialize
-        # Use session to send to backend
-        pass
+        url = construct_url(
+            base_url=self._ge_cloud_base_url,
+            organization_id=self._ge_cloud_organization_id,
+            resource_name="migration",
```

Comment:
The resource name, endpoint name, and actual payload structure seems to be TBD? These are just placeholders but should eventually utilize the enum defined in `GeCloudStoreBackend`?

Reply:
This will have to be provided by the cloud backend team. I agree it makes sense to add this to the enum.